tests/integration/zenodo_test.py

```diff
@@ -226,16 +246,31 @@ async def identity(x):
         )
     )
 
-    v2 = await orchestrator.run()
-    v2_refreshed = await orchestrator.depositor.get_record(v2.id_)
+    # Should fail due to deleted file
+    v2_summary = await orchestrator.run()
```

Comment:
Is the idea that this will remove a file, and `check_missing_files` says "if there was a file before, and now that file is gone, SCREAM"?

Reply:
Yeah the `check_missing_files` test will be enabled by default since that would generally indicate something like a missing year, so this will first check that it failed, then switch the test off and check that it succeeds the second time.